nonCisMaleInCs/src/components/Details.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import {
+  StyleSheet,
+  View,
+  Text,
+  Image,
+  Alert,
+  TouchableOpacity,
+  ScrollView,
+} from 'react-native';
+import editIcon from '../images/edit.png';
+import nophoto from '../images/nophoto.png';
+
+import {Colors} from 'react-native/Libraries/NewAppScreen';
+
+const onPressed = () => {
+  Alert.alert('Edit button got pressed!');
+};
+
+const Details = () => {
```

Comment:
we have arrow function here. another place we have the following:

```
function Dashboard() {}
```
another place we have 
```
export default function Card({item}) {}
```
can we be consistent here too? :) 
- arrow functions, yes or no
- export default syntax in the beginning or in the end

Reply:
@princiya all corrections are done! I changed all functions to arrow functions and added export statements at the bottom of the component. Please review it 😄